src/ASM/ASMs3Dmx.C

```diff
@@ -489,7 +489,8 @@ bool ASMs3Dmx::integrate (Integrand& integrand,
       Matrix3D Hess;
       double dXidu[3];
       Matrix Xnod, Jac;
-      Vec4   X;
+      double   param[3] = { 0.0, 0.0, 0.0 };
+      Vec4   X(param);
```

Comment:
Ah, so it seems you did implement it for mixed 3D. Then I guess the param-stuff was just forgotten for ASMs2Dmx?

Reply:
done.